src/main/webapp/resources/js/apis/clients/clients.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+/**
+ * Revoke all tokens for a given client by identifier.
+ * @param {number} id for the client
+ * @return {Promise<AxiosResponse<T>>}
+ */
+export async function revokeClientTokens(id) {
+  return await axios.put(`clients/ajax/revoke?id=${id}`);
```

Comment:
Is this URL going to work in a context path?

Reply:
Yes, it gets handled automatically by as the request leaves.   You can find the code for that on line 71 of `app.js`.  Developers do not need to set the context path now (but it does not fail if they do).